src/tools/publish_toolstate.py

```diff
@@ -202,7 +202,7 @@ def update_latest(
                 old = status[os]
                 new = s.get(tool, old)
                 status[os] = new
-                maintainers = ' '.join('@'+name for name in MAINTAINERS[tool])
+                maintainers = ' '.join('@'+name for name in MAINTAINERS.get(tool, ())
```

Comment:
```suggestion
                maintainers = ' '.join('@'+name for name in MAINTAINERS.get(tool, ()))
```

Reply:
I think this should fix another failure, hopefully..

Reply:
Oh, original PR has the same fix :)
@Xanewok You will continue the work here, or on the original?

Reply:
@JohnTitor thanks for spotting this as well! I'll continue there to land this asap for everyone else